Cargo.toml

```diff
@@ -22,10 +22,10 @@ router = "0.1.0"
 rustc-serialize = "0.3"
 serde = "0.6.13"
 serde_json = "0.6.0"
-serde_macros = "0.6.13"
+serde_macros = "0.6.14"
 staticfile = "0.1.0"
 uuid = "0.1.18"
-ws = "0.4.5"
+ws = { git = "https://github.com/hfiguiere/ws-rs.git" }
```

Comment:
We can switch back to upstream https://github.com/housleyjk/ws-rs.git since they merged your fixes.